.github/workflows/ci.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       matrix:
         platform: [ubuntu-latest, windows-latest]
-        python-version: ['3.6', '3.7', '3.8', '3.9', '3.10', '3.11-a05']
+        python-version: ['3.6', '3.7', '3.8', '3.9', '3.10', '3.11.0-alpha.5']
```

Comment:
Is it possible to specify `'3.11' here to always get the latest available development version?

Reply:
Updated to use 3.11